parts/linux/cloud-init/artifacts/ubuntu/cse_install_ubuntu.sh

```diff
@@ -80,6 +80,13 @@ cleanUpGPUDrivers() {
     rm -Rf $GPU_DEST /opt/gpu
 }
 
+installCriCtlPackage() {
+    version="${1:-}"
```

Comment:
what happens if we default to empty string here?

Reply:
hmm. Good point. Let me spin up a VM to check.

Reply:
exit with error if version is empty because it's required in components.json